autoscheduler/scraper/views.py

```diff
@@ -14,3 +16,16 @@ def get_object(self):
         course_num = self.request.query_params.get('course_num')
         term = self.request.query_params.get('term')
         return Course.objects.get(dept=dept, course_num=course_num, term=term)
+
+class RetrieveSectionView(generics.RetrieveAPIView):
+    """ API endpoint for viewing course information, used by /api/sections.
+        This view returns a serialized section, should return .
+    """

```

Comment:
This comment should be changed to be more descriptive/accurate (returns serialized list of sections)